src/components/schemas/fields.yaml

```diff
@@ -1507,3 +1507,18 @@ query_params:
   description: | 
     This field contains the query param string similar to the query params passed in GET call for displaying data in the list view.
   example: state=eq.PAID
+
+joined_at:
+  type: string
+  nullable: true
+  description: |
+    Date and time when the user joined the organization.
+  example: '2020-06-11T13:14:55.201598+00:00'
+
+mobile:
```

Comment:
we have already defined mobile field in employees 
```   mobile:
      type: string
      nullable: True
      items:
        $ref: './fields.yaml#/phone_number'
      description: |
        Mobile number of the employee
   ```
   
   can we not use the `#/phone_number` ref here?

Reply:
oh using it then